ledger/apply/application_test.go

```diff
@@ -962,6 +963,67 @@ func TestAppCallApplyUpdate(t *testing.T) {
 	a.Equal([]byte{2}, br.AppParams[appIdx].ApprovalProgram)
 	a.Equal([]byte{2}, br.AppParams[appIdx].ClearStateProgram)
 	a.Equal(basics.EvalDelta{}, ad.EvalDelta)
+
```

Comment:
add a test case checking a current (base) behavior: update failure on `ConsensusV28` when extra pages are used in app params.

Reply:
I don't think this behavior has changed. we are still only allowing ExtraProgramPages to be set at app creation.

Reply:
the problem is this behavior was not tested, and now we are making a protocol upgrade change. So a base behavior needs to be checked to ensure the difference.

Reply:
ok. Added this case to transaction_test